index.bs

```diff
@@ -636,6 +636,9 @@ delay to deliver an [=aggregatable report=].
 <dfn>Randomized aggregatable report delay</dfn> is a positive [=duration=] that controls the
 random delay to deliver an [=aggregatable report=].
 
+<dfn>Default aggregation coordinator</dfn> is the [=aggregation coordinator=] that controls how to
```

Comment:
I think it should be fine to just define this constant in the spec vs. leaving it as vendor specified. Otherwise, my preference would be to avoid a default in the first place.

Reply:
It seems a bit strange if the allowed list is vendor specific but the default is not, so just want to confirm whether we want to define all these constants or just the default.

Reply:
I think it should be OK to pull all these out into constants. If needed in the future we can have an auxiliary list of vendor defined allowed values but I don't think we need to right now.

Reply:
That makes sense, updated, PTAL, thanks!

Reply:
@csharrison Could you please take another look? Thanks.